pyproject.toml

```diff
@@ -9,6 +9,7 @@ requires = [
     "numpy==1.15.4; python_version>='3.7' and platform_system!='AIX'",
     "numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'",
     "numpy==1.16.0; python_version>='3.7' and platform_system=='AIX'",
+    "numpy==1.17.3; python_version>='3.8' and platform_system=='AIX'",
```

Comment:
I think you should also change the line directly above, right now both rules will apply (`>=3.7` and `>=3.8`)

Reply:
Does that matter, or is the tightest constraint picked? I'm copying this from scipy (https://github.com/scipy/scipy/blob/master/pyproject.toml) who I trust to understand this better than we do.

Reply:
I'm not sure, PEP-518 (where the format of `pyproject.toml` is described) points to PEP-508 which only says that anything that evaluates to `False` is ignored. Anyways, scipy does have `python_version==3.7` instead of `python_version>=3.7`, so I guess you should copy that?

Reply:
Good catch, thanks.